### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by Rad586**

### CODE REVIEW

**Comment by tristankechlo** [comment]
First of all, thanks for taking the time translating the mod!  
  
Now I got a few points:
 - If you want to be credited, also add your name to the Translations part of the CREDITS.md
 - Are you ok with me, bringing the translation to other modversions?

File: Common/src/main/resources/assets/livingthings/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
+
+  "itemGroup.livingthings.general":"鲜活生命",
```
Comment (tristankechlo) [inline Common/src/main/resources/assets/livingthings/lang/zh_cn.json]: Can you reverse this line to `Living Things`, I like to keep the item-group name consistent between all languages

**Comment by Rad586** [comment]
> First of all, thanks for taking the time translating the mod!
> 
> Now I got a few points:
> 
> * If you want to be credited, also add your name to the Translations part of the CREDITS.md
> * Are you ok with me, bringing the translation to other modversions?

Sure, feel free to do whatever you want :)

**Comment by tristankechlo** [comment]
Thank you for the translation

**Pull request merged**